playbooks/adjust-cinder-backend.yml

```diff
@@ -47,6 +47,12 @@
       when: not cinder.enabled|default('False')|bool
       run_once: true
 
+    - fail: msg="Only one type of osds should be configured"
```

Comment:
Also this looks like an unrelated change, please account for it in the commit message(s).